pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -714,7 +714,7 @@ async def _build_images(self):
         successful_build_nvrs = list(filter(lambda x: x, [build.get("nvrs") for build in success_map.values()]))
 
         if successful_build_nvrs:
-            jenkins.start_scan_osh(build_nvrs=successful_build_nvrs)
+            jenkins.start_scan_osh(nvrs=successful_build_nvrs)
```

Comment:
I'm thinking how much delay is there between rpm and image builds, and if it matters to trigger 2 jobs, rather than one job at the end for all built nvrs?

Reply:
Good idea, so a class variable to keep track of all the NVRs and then trigger it once?

Reply:
Added in https://github.com/openshift-eng/aos-cd-jobs/pull/3917/commits/3803c412739aa1588e2942a2c5f569157adf8ce3